app/views/tweets/tweets.erb

```diff
@@ -0,0 +1,5 @@
+<h3>Welcome, <%= current_user.username %></h3>
+
+<% Tweet.all.each do |tweet| %>
```

Comment:
MVC design pattern dictates that interactions with `models` should be done in the controller.